frontstage/views/errors.py

```diff
@@ -18,11 +18,14 @@ def default_error_page():
 
 @error_bp.route('/404', methods=['GET', 'POST'])
 def not_found_error_page():
-    response = make_response(render_template('errors/404-error.html', _theme='default', data={"error": {"type": "failed"}}))
-    return response
+    return render_template('errors/404-error.html', _theme='default', data={"error": {"type": "failed"}}), 404
 
 
 @error_bp.route('/500', methods=['GET', 'POST'])
 def server_error_page():
-    response = make_response(render_template('errors/500-error.html', _theme='default', data={"error": {"type": "failed"}}))
-    return response
+    return render_template('errors/500-error.html', _theme='default', data={"error": {"type": "failed"}}), 500
+
+
+@error_bp.route('/403', methods=['GET', 'POST'])
+def not_logged_in_error_page():
+    return render_template('errors/not-signed-in.html', _theme='default', data={"error": {"type": "failed"}}), 403
```

Comment:
Will put them in order. 

Not sure how many more we'll need, these are all the cases I'd think we'd want for MVP